example_scripts/coupling_no_mpda/MainCouplingNoMdpaParticles.py

```diff
@@ -0,0 +1,270 @@
+""" This files provide a test example on how to use
+Simphony for coupling CFD with DEM
+
+"""
+import os
+import math
+
+from simphony.api import CUDS, Simulation
+from simphony.core.cuba import CUBA
+from simphony.cuds.meta import api
+
+from simphony.core.data_container import DataContainer
+from simphony.cuds.particles import Particles as SParticles
+from simphony.cuds.particles import Particle as SParticle
+
+# This line is optional and only adds some tools to read kratos models
+# as simphony datasets rather than crating them in the script.
+from simkratos.CFD.kratos_CFD_utils import CFD_Utils
+
+
+def simple_coupling(cuds, particles_dataset_name):
+    viscosity = 1.0e-3
+    something_shaddy = 6.0
+    particle_dataset = cuds.get_by_name(name=particles_dataset_name)
+
+    for particle in particle_dataset.iter(item_type=CUBA.PARTICLES):
+        p_data = particle.data
+
+        vel = p_data[CUBA.VELOCITY]
+        rel_vel = p_data[CUBA.RELATIVE_VELOCITY]
+        radius = p_data[CUBA.RADIUS]
+
+        factor = something_shaddy * math.pi * viscosity * radius
+
+        external_force = [factor * (x - y) for x, y in zip(rel_vel, vel)]
+
+        particle.data[CUBA.EXTERNAL_APPLIED_FORCE] = external_force
+        particle_dataset.update([particle])
+
+
+def generate_particles(smp_particles, smp_conditions, smp_materials, smp_pe):
+    # Define the particle containers.
+    particles = SParticles(name='particles')
+
+    # Fill the data ( particle )
+    data = DataContainer()
+    data[CUBA.RADIUS] = 0.05
+
+    particles.add([
+        SParticle(
+            coordinates=(0.2, 0.4, 0.4),
+            data=DataContainer(data),
+        )
+    ])
+
+    material = api.Material(name='material_' + particles.name)
+    materialData = material.data
+
+    materialData[CUBA.DENSITY] = 2500.0
+    materialData[CUBA.YOUNG_MODULUS] = 35e9
+    materialData[CUBA.POISSON_RATIO] = 0.20
+    materialData[CUBA.FRICTION_COEFFICIENT] = 0.5773502691896257
+    # materialData[CUBA.PARTICLE_COHESION] = 0.0
+    materialData[CUBA.RESTITUTION_COEFFICIENT] = 0.02
+    materialData[CUBA.ROLLING_FRICTION] = 0.01
+    # materialData[CUBA.DEM_CONTINUUM_CONSTITUTIVE_LAW_NAME] = "DEM_KDEMFabric"
+    # materialData[CUBA.DEM_DISCONTINUUM_CONSTITUTIVE_LAW_NAME] = "DEM_D_Hertz_viscous_Coulomb"
+    # materialData[CUBA.CONTACT_TAU_ZERO] = 25
+    # materialData[CUBA.CONTACT_SIGMA_MIN] = 5
+    # materialData[CUBA.CONTACT_INTERNAL_FRICC] = 1
+
+    material.data = materialData
+
+    particlesData = particles.data
+    particlesData[CUBA.MATERIAL] = material.name
+    particles.data = particlesData
```

Comment:
@khiltunen This is the way to change the properties.
As you can see not all of them are enabled, so if they have no entry point in the CUBA keywords they will use the default value.

The wrapper is prepared so as long as soon as they can be stored in CUBA they can be enabled just adding its name in the wrapper.

If some of the properties commented are needed for your simulations, pleas tell me so we can ping someone who can update the oficial CUBA keyword list ( or make a PR requesting the changes in the simphony-metadata repo)